ui/webui/webui-desktop

```diff
@@ -97,28 +61,17 @@ case "$1" in
         ;;
 esac
 
-detect_browser
-
-# start the bridge; this needs to run in the normal user session/namespace
-coproc ${2:+ssh "$2"} cockpit-bridge
-trap "kill $COPROC_PID; wait $COPROC_PID || true" EXIT INT QUIT PIPE
+BROWSER="/usr/bin/firefox --kiosk"
```

Comment:
So I'm using this variable on my Firefox theme PR: https://github.com/rhinstaller/anaconda/pull/4918/files#diff-4384704581bd02e11beb389b584be16b175b80679194ac91422ed2701d8bd3bdR139

I wonder - now that we started changing it maybe this script is a better place for the theme handling than the UI handling Python code where I put it in my PR ? What do you think ? 

We might have to solve live/boot.iso detection somehow, but otherwise it should work.

Reply:
Yes it probably makes more sense that the theming gets applied here.

Reply:
OK, so I'll adjust https://github.com/rhinstaller/anaconda/pull/4918 accordingly. :)